live-examples/js-examples/regexp/regexp-groups-ranges.js

```diff
@@ -1,10 +1,14 @@
-const aliceExcerpt = 'The Caterpillar and Alice looked at each other';
-const regexpWithoutE = /\b[a-df-z]+\b/gi;
-console.log(aliceExcerpt.match(regexpWithoutE));
-// Expected output: Array ["and", "at"]
-
+// Groups
 const imageDescription = 'This image has a resolution of 1440×900 pixels.';
 const regexpSize = /([0-9]+)×([0-9]+)/;
 const match = imageDescription.match(regexpSize);
 console.log(`Width: ${match[1]} / Height: ${match[2]}.`);
 // Expected output: "Width: 1440 / Height: 900."
+
+// Backreferences
+const quote = `Single quote "'" and double quote '"'`;
+const regexpQuotes = /(['"]).*?\1/g;
+for (const match of quote.matchAll(regexpQuotes)) {
+  console.log(match[0]);
+}
+// Expected output: '"'"' '"'"'
```

Comment:
Let's use an example that's not so confusing with so many nested quotes. What about the [duplicate words](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Regular_expressions/Backreference#matching_duplicate_words) example instead?